### ISSUE TITLE
Add Accessibility Statement page to the UGRC website

### ISSUE THREAD

**Issue by chriswnek** [agrc/planning-queue#960]
To show transparency in the efforts being put forth by UGRC to meeting accessibility requirement, we will add a page to the UGRC website that includes an Accessibility Statement, along with a link to our Accessibility Action Plan document.  Where necessary, we can link to this page from other websites or web apps that UGRC manages.

### Tasks
- [x] Draft Accessibility Statement for website
- [x] Draft Accessibility Action Plan (In progress)
- [x] Create new page on website
- [x] Add link to page in the footer of gis.utah.gov

#### Accessibility page text
UGRC is committed to ensuring digital accessibility for all users, including those interacting with our geospatial information systems. We believe that location-based data should be inclusive. We strive to provide an equitable experience that aligns with [WCAG 2.1 Level AA](https://www.ada.gov/resources/2024-03-08-web-rule/#:~:text=Requirement:%20State%20and%20local%20governments'%20mobile%20apps%20usually%20need%20to,run%20by%20a%20private%20company) standards for all websites and applications that we maintain. If a UGRC website or application does not meet your accessibility needs, please contact us at ugrc@utah.gov.

We created the [Accessibility Action Plan](https://docs.google.com/document/d/e/2PACX-1vS-Y54mTB9D7H1rVk4xmZa-8TrCitVwrE0HihpTdlQ_y6k7na45mixr9CQhMrf7R4ZxvIWQ2fLeUUWI/pub) to serve as our formal roadmap for identifying and remediating digital barriers across our websites and GIS applications. We maintain this as a living document to outline our testing procedures and remediation workflows, ensuring a consistently inclusive experience for you.

**Comment by chriswnek** [comment]
I've worked on modifying our AAP Google Doc to address accessibility issues that it was having when published to the web.  Fixes included removing the tabbed sections and instead making it one long document, adding padding around links, and fixing table headers.

WAVE can't seem to handle analyzing the published Google Doc, but Lighthouse does work.  Only two issues remain:

- _<html> element does not have a [lang] attribute_: the html of a published Google Doc cannot be modified, so there is not workaround for this issue
- _Document does not have a main landmark_: similar to the above language error, where the issue is with Google's html template.  A recommended workaround was to add a Table of Contents to the top of the document to help a screen reader with navigation, so that's been implemented.  Another "band-aid" we can do is to add a `role` and `aria-label` to the iframe code.  Below is an example of what that would look like:

```
<iframe 
src="https://docs.google.com/document/d/e/2PACX-1vS-Y54mTB9D7H1rVk4xmZa-8TrCitVwrE0HihpTdlQ_y6k7na45mixr9CQhMrf7R4ZxvIWQ2fLeUUWI/pub?embedded=true"
role="main"
title="Main Document Content"
aria-label="Main Document Content">
</iframe>
```

**Comment by chriswnek** [comment]
Modified the draft accessibility text so it better followed our UGRC style guide.

**Comment by chriswnek** [comment]
@stdavis The content for the new accessibility page on our website is ready to be published.  We never discussed the page URL though.  I'm not sure if we have rules about the hierarchy? For example should we go with gis.utah.gov/accessibility or should it be under "About", so gis.utah.gov/about/accessibility?  cc: @leault

**Comment by stdavis** [comment]
I vote for it to be nested under `/about`. If you want `gis.utah.gov/accessibility` to work, we can add a redirect for that.

**Comment by chriswnek** [comment]
`/about` works for me. No need for a redirect.

**Comment by stdavis** [comment]
This content has been added to gis.utah.gov.

**Comment by stdavis** [comment]
FYI. We may not be able to link to our accessibility page after we switch to the UDS footer.

<img width="626" height="262" alt="Image" src="https://github.com/user-attachments/assets/a71bd2b6-9310-4e48-9d31-745e126cf0e1" />
